packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -0,0 +1,211 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/actiongroup';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import Brush from '@spectrum-icons/workflow/Brush';
+import Edit from '@spectrum-icons/workflow/Edit';
+import {Item} from '@react-stately/collections';
+import RegionSelect from '@spectrum-icons/workflow/RegionSelect';
+import Select from '@spectrum-icons/workflow/Select';
+```
+
+---
+category: Buttons
+---
+
+# ActionGroup
+
+<p>{docs.exports.ActionGroup.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionGroup']} />
+
+## Example
+
+```tsx example
+<ActionGroup>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+## Content
+
+ActionGroups accept as list of `children`, each with a `uniqueKey` prop. Generally `children` will be `Item` components from the `@react-stately/collections` library.
+
+However, a function that returns `Item` components is also accepted.
+
+```tsx example
+const items = [
+  {children: 'React', name: 'React'},
+  {children: 'Add', name: 'Add'},
+  {children: 'Delete', name: 'Delete'}
+];
+<ActionGroup items={items} itemKey="name">
+  {item => <Item textValue={item.name}>{item.children}</Item>}
+</ActionGroup>
+```
+
+
+In order to identify children when a user takes an action on a item, each child needs to have a `uniqueKey` prop. If using a function, an `itemKey` prop is required on the ActionGroup.
```

Comment:
Maybe "Each child needs to have a `uniqueKey` prop so it can be identified when actioned upon by a user."?